application/repositories.go

```diff
@@ -49,6 +50,9 @@ type SearchRepository interface {
 // IdentityRepository encapsulates identity
 type IdentityRepository interface {
 	List(ctx context.Context) (*app.IdentityArray, error)
+	Load(ctx context.Context, ID string) (*app.Identity, error)
```

Comment:
Don't rely on app.* on the Repository level. app is the external API. account.Identity is the internal.